src/components/pluginManager.js

```diff
@@ -74,14 +74,23 @@ import { playbackManager } from './playback/playbackmanager';
             if (typeof pluginSpec === 'string') {
                 if (pluginSpec in window) {
                     console.log(`Loading plugin (via window): ${pluginSpec}`);
-                    let pluginInstance = await window[pluginSpec];
 
-                    if (typeof pluginInstance === 'function') {
-                        pluginInstance = await new pluginInstance();
+                    const pluginDefinition = await window[pluginSpec];
+                    if (typeof pluginDefinition !== 'function') {
+                        const err = new TypeError('Plugin definitions in window have to be an (async) function returning the plugin class');
```

Comment:
Doesn't need to be async. A normal function should also work fine.

Reply:
That's why it's in parentheses, both are supported.